index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine|Inconsolata|Droid+Sans|Poppins">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700:900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+    <title>Porfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header id="header"> 
+            <div class="logo blue-text">Joeveloper</div>
+            <nav class="nav-bar">
+                <ul id="navlink-ul">
+                    <li><a class="nav-links black-text" href="">Portfolio</a></li>
+                    <li><a class="nav-links black-text" href="">About</a></li>
+                    <li><a class="nav-links black-text" href="">Contact</a></li>
+                    <li><a class="nav-links black-text" href=""><i class="fas fa-envelop"></i></a></li>
+                </ul>
+                <div class="burger-parents">
+                    <div class="hamburger"></div>
+                    <div class="hamburger"></div>
+                    <div class="hamburger"></div>
+                </div>
+                
+        </nav>
+    </header>
+
+    <div id="portfolio-main">
+        <div id="div">
+            <div class="main-intro">
+                <div id="maintext-body">
+                    <h1 class="intro-heading .black-text">I'm Joeveloper</h1>
+                    <h1 class="intro-heading">Glad to see you!</h1>
+                    <p id="intro-para" class="black-text">I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+                </div>
```

Comment:
Kindly ensure that the title and the paragraph uses the same font-size and color as the figma design to in order to match the original design since the purpose of this activity is to produce a design as close to the original one as possible. 😉